controllers/component_build_controller_test.go

```diff
@@ -1459,6 +1653,7 @@ var _ = Describe("Component initial build controller", func() {
 				},
 			}
 			Expect(k8sClient.Create(ctx, selectors)).To(Succeed())
+			getComponent(resourceKey)
```

Comment:
Why do we need to wait for component after creation of the selectors?

Reply:
mistake, removed :)

Reply:
and added to another